test/task_helper_functions.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function generate() {
+  pip3 install --user -r /workspace/helpers/generate_modules/requirements.txt
+  /workspace/helpers/generate_modules/generate_modules.py
+}
+
+function check_generate() {
```

Comment:
@paulpalamarchuk I still don't understand why this differs so much from master. It's not clear to me why this would need to be different from the logic we already have [written](Right, what I don't understand is why the logic for this function is so different from what's currently implemented on [master](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/master/test/make.sh#L164). Instead of needing to ).

Reply:
@morgante I changed this function to avoid errors on CI, in case of using `git-diff` to compare files. Error: `fatal: not a git repository (or any parent up to mount point /)`

Reply:
Ok it's workable then.